### ISSUE TITLE
[BUG] Unsupported course template

### ISSUE THREAD

**Issue by oxfighterjet** [FallingLights/Teachable-dl#54]
Thank you for working on Teachable-dl, it's a worthwile project (the only one still being worked on for teachable it seems).

**Describe the bug**
It seems I've encountered a course template which is unsupported:
https://bouqpaperflowers.teachable.com/courses/enrolled/1661064

I tried the following:
`python3 main.py --url https://bouqpaperflowers.teachable.com/courses/enrolled/1661064 --man_login_url https://sso.teachable.com/secure/319838/identity/login/otp --verbose`

This opened up the chromium browser. I entered my email address and hurried up to enter the TOTP sent to that email address to login. After that, I got the following terminal output:

```
INFO: Starting login
INFO: Waiting for user to navigate to url: https://sso.teachable.com/secure/319838/identity/login/otp
INFO: Current url: https://sso.teachable.com/secure/319838/identity/login/otp
INFO: Starting download of course: https://bouqpaperflowers.teachable.com/courses/enrolled/1661064
INFO: Switching to course page
INFO: Picking course downloader
ERROR: Downloader does not support this course template. Please open an issue on github.
INFO: Cleaning up
```

**Desktop (please complete the following information):**
 - OS: Fedora 41
 - Chromium: Version 131.0.6778.204 (Official Build) Fedora Project (64-bit)

If there's any more information I can help with, please let me know. Thank you!

**Comment by brettthomson** [comment]
I am getting this error also. As when i enter the OTP it lands me on the page and I am unable to click on my course before getting the same above error.

This is the only project that works. I really hope someone can update it to resolve this

**Comment by FallingLights** [comment]
To resolve this issue, I would need access to the courses, as I currently do not have access to any Teachable course.

**Comment by oxfighterjet** [comment]
Thank you for your response!

> To resolve this issue, I would need access to the courses, as I currently do not have access to any Teachable course.

What do you mean you do not have access to any teachable course?

The course I [linked](https://bouqpaperflowers.teachable.com/courses/enrolled/1661064) to above is free. You can create a dedicated new throwaway teachable account and get enrolled without having to pay.

**Comment by riprip74** [comment]
I tried the link you posted, and it works for me.
I entered the following command:
<code class="no-translate">
"py main.py --url https://bouqpaperflowers.teachable.com/courses/enrolled/1661064 --man_login_url https://bouqpaperflowers.teachable.com/courses/enrolled/1661064 --verbose"
</code>
I put the same url for --man_login_url
It let me plenty of time to login and then download starts when I reach the url https://bouqpaperflowers.teachable.com/courses/enrolled/1661064

**Comment by drackp2m** [comment]
Indeed, the key is to put the same URL in the url argument and in the man_login_url as well. I used login by email / password, then I manually accessed the course URL, and everything started to work.
